redaxo/src/core/fragments/core/form/searchfield.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+$id = '';
+$class = 'input-group input-group-xs has-feedback form-clear-button';
+$clear = rex_i18n::msg('search_clear');
+$placeholder = ' placeholder = "' . rex_i18n::msg('search_placeholder') . '"';
```

Comment:
```suggestion
$placeholder = ' placeholder="' . rex_i18n::msg('search_placeholder') . '"';
```

Besser wäre jedoch, die Attribute in einem Array zu legen und dann via `rex_string::buildAttributes($array)` ausgeben